lib/loader.js

```diff
@@ -32,13 +32,42 @@ function getMatches (type, items, matches, component) {
   return imports
 }
 
-function install (install, content, imports) {
+function injectStylesSSR (imports) {
+  const styles = imports.map(componentImport => (componentImport[2] || [])).reduce((acc, styles) => {
+    styles && styles.forEach(style => acc.add(style))
+    return acc
+  }, new Set())
+
+  if (styles.size) {
+    return `
+if (process.server) {
```

Comment:
Nuxt specific? The vue ssr docs say to use `process.env.VUE_ENV`, which is also available in nuxt.

Reply:
Might be, will change 👍

Reply:
Fixed!